storeshare_api/spec/rails_helper.rb

```diff
@@ -0,0 +1,88 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+require 'devise'
+require_relative 'support/controller_macros'
+
+ENV['RAILS_ENV'] ||= 'test'
+
+require File.expand_path('../config/environment', __dir__)
+
+# Prevent database truncation if the environment is production
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'rspec/rails'
+# Add additional requires below this line. Rails is not loaded until this point!
```

Comment:
once the code is more final we should probably remove the commented-out boilerplate code